cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -761,6 +761,24 @@ static PyObject *get_precision(PyObject *self, PyObject *args)
     return PyLong_FromLong(precision);
 }
 
+static PyObject *get_simulator_product(PyObject *m, PyObject *args)
+{
+    COCOTB_UNUSED(m);
+    COCOTB_UNUSED(args);
+    const char *product_string = gpi_get_simulator_product();
+    PyObject *result = Py_BuildValue("s", product_string);
+    return result;
+}
+
+static PyObject *get_simulator_version(PyObject *m, PyObject *args)
+{
+    COCOTB_UNUSED(m);
+    COCOTB_UNUSED(args);
+    const char *version_string = gpi_get_simulator_version();
+    PyObject *result = Py_BuildValue("s", version_string);
+    return result;
```

Comment:
Don't think extra lines offer any clarity here:
```suggestion
    return PyUnicode_FromString(gpi_get_simulator_version());
```